custom/enikshay/private_sector_datamigration/models.py

```diff
@@ -480,6 +484,34 @@ class Adherence(models.Model):
     unknwDoseReasonId = models.CharField(max_length=8, null=True)
 
     @property
+    def adherence_report_source(self):
+        return {
+            REPORTING_MECHANISM_MISSED_CALL: 'missed_call',
+            REPORTING_MECHANISM_99_DOTS: '',
+            REPORTING_MECHANISM_FIELD_OFFICER: 'field_officer',
+            REPORTING_MECHANISM_TREATMENT_SUPPORTER: 'treatment_supervisor',
```

Comment:
why is this `TREATMENT_SUPPORTER` when the case property is `treatment_supervisor`?